statchatui/src/Services/api.js

```diff
@@ -0,0 +1,33 @@
+import { GAME_NAMES } from "../Data/GameNames";
+
+const fetchStats = async (formData) => {
+  const apexApiKey = process.env.REACT_APP_API_KEY;
+  const baseUrl = "https://api.mozambiquehe.re/bridge?auth=";
+  if (
+    formData.gamertag === "" ||
+    formData.platform === "" ||
+    formData.gameName === ""
+  ) {
+    return null;
+  }
+  try {
+    let response;
+    if (formData.gameName === GAME_NAMES.APEX) {
+      response = await fetch(
+        `${baseUrl}${apexApiKey}&player=${formData.gamertag}&platform=${formData.platform}`
+      );
+    } else if (formData.gameName === GAME_NAMES.R6) {
+      response = await fetch(
+        `https://api.henrikdev.xyz/r6/v1/profile/${formData.platform.toLowerCase()}/${
+          formData.gamertag
+        }`
+      );
+    }
+    const data = await response.json();
+    return data;
+  } catch (error) {
+    console.error("Error submitting form:", error);
```

Comment:
It looks like we catch the error here. That means that if something goes wrong, users will seee the form silently fail.

It might be better if we had the React component catch the error instead.

You can re-throw the error here: 
```suggestion
    console.error("Error submitting form:", error);
    throw error;
```

(don't forget you also need to handle it if you re-throw it).

Reply:
This might need to be done in a different PR, although I vaguely remember that you implemented error handling at some point in the past.